app/serializers/articles/show_serializer.rb

```diff
@@ -2,6 +2,6 @@ class Articles::ShowSerializer < ActiveModel::Serializer
   attributes :id, :title, :teaser, :content, :category, :premium
 
   def content
-    object.premium ? object.content[0...100] : object.content
+    object.premium ? object.content[0...299] : object.content
```

Comment:
You need a conditional that sends the whole content if the current user is subscriber